config.php

```diff
@@ -0,0 +1,6 @@
+<?php
+$config = [
```

Comment:
This is not a requirement, but if you split  your config under it should be more maintainable like : $config['core'], $config['database'], $config['view']